site/en/blog/chips-origin-trial-extended/index.md

```diff
@@ -2,13 +2,14 @@
 layout: 'layouts/blog-post.njk'
 title: Cookies Having Independent State (CHIPS) origin trial extended
 description: >
-  CHIPS is a Privacy Sandbox proposal that introduces a mechanism to opt-in
-  to having third-party cookies partitioned by top-level site. The origin trial
-  that started in Chrome 100, will now be available until the release of Chrome 105,
-  scheduled for August 30, 2022.
+  CHIPS is a Privacy Sandbox proposal that introduces a mechanism to opt-in to
+  having third-party cookies partitioned by top-level site. The origin trial
+  that started in Chrome 100, will now be available until October 25, 2022
+  around the end of the Chrome 106 Stable release.
```

Comment:
Just to clarify what this means here--the OT will be available till Oct 25. It started with Chrome 100 and will be available till 106 Stable release? Bit confused by the phrasing regarding the "end of".

Reply:
I think the updated wording removes this now.